app/models/user.rb

```diff
@@ -45,4 +45,8 @@ def get_random_task_no_categories(exclude: nil)
   def completed_tasks
     self.tasks.where(is_multi_use: false)
   end
+
+  def skillList
```

Comment:
I really don't know the reason, but the Ruby community prefers underscores to camel case. I heard it's because there a lot of Ruby programmers that are not native speakers, so they find it more readable, but who knows!